framework/core/events.ts

```diff
@@ -375,6 +375,13 @@ export class EventEmitter {
     if (n !== Infinity) {
       if (n === 0) {
         n = Infinity
+      } else {
```

Comment:
In the Deno repo's `events.ts`, there is a call here to validation code in a utility module. Rather than adding another dependency to Aleph, I just inlined the logic here.

Reply:
great! the `events.ts` is copied from `deno.land/std` for sure! i removed the deps since the client(browser) needs it too.